.github/workflows/tox-test.yml

```diff
@@ -0,0 +1,103 @@
+name: Tox tests
+
+on: [push, pull_request]
+
+jobs:
+  py27:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Intsall RPM
+        run: sudo apt-get install -y rpm
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 2.7
+      - name: Install Tox
+        run: pip install tox
+      - name: Run Tox
+        run: tox -e py27
+  static:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Intsall RPM
+        run: sudo apt-get install -y rpm
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
```

Comment:
I suggest bumping all these 3.6 to 3.8, as mentioned above I don't think we need to support any 3.x older than 3.8 now.

Reply:
Updated workflows for Python 3.8